core/network/types/state_request.hpp

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_STATE_REQUEST_HPP
+#define KAGOME_STATE_REQUEST_HPP
+
+#include "common/buffer.hpp"
+#include "primitives/common.hpp"
+
+namespace kagome::network {
+  struct StateRequest {
+    // Block header hash.
```

Comment:
It must have triple-slash to be doc-comment

Reply:
Done